.storybook/themes/commerce.scss

```
@@ -334,6 +334,46 @@
 					max-width: var(--content-scale-width),
 					width: 100%,
 				),
+				carousel-indicator-dots-container: (
+					display: flex,
+					gap: var(--global-spacing-4),
+					padding: var(--global-spacing-4) 0 0 0,
+					justify-content: center,
+					align-items: center,
+				),
+				carousel-indicator-dot: (
+					height: var(--global-spacing-3),
+					width: var(--global-spacing-3),
+					border: var(--global-border-width-1) solid var(--border-color),
+					border-radius: var(--border-radius-circle),
+					background-color: transparent,
+				),
+				carousel-indicator-dot-active: (
+					border: none,
+					height: var(--global-spacing-4),
+					width: var(--global-spacing-4),
+					background-color: var(--global-neutral-4),
+				),
+				carousel-indicator-thumbnails-container: (
+					padding: var(--global-spacing-4) var(--global-spacing-3) 0 var(--global-spacing-3),
+					width: 100%,
+					overflow-x: scroll,
```

If we use `auto` this will only place the scroll bar when needed and we avoid having the default browser UI for scrollbar showing at all times

```suggestion
					overflow-x: auto,
```

---

cool good improvement